docs/blockchain/contracts/nft-contract/nft-actions/mkfctofr.a.experimental.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'mkfctofr.a'
+order: 40
+
+---
+
+# mkfctofr.a
+
+Make a buy offer on a given factory.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the factory.
+
+The action stores the offer to `fctoffer.a` table with the specified arguments. The new offer ID is read from `next.fctofr` table whose value field is then incremented.
+
+`eosio.nftram` pays RAM usage.
+
+The offered price will be transferred to `eosio.nftofr` account and will be kept until the offer is either accepted or cancelled.
+
+An account will not be able to make offers on the same factories that they already made.
+
+An account will not be able to make more offers on factories than `max_active_offer_per_user` of `offercfg.a`, which records the global configurations.
```

Comment:
Should be 
An account will not be able to make more offers on either factories or tokens.

Reply:
No need for update here.
This action page is about making an offer on uniq factory, so this is true in this case.
Same statement made on making offer on NFT page but replaced with NFTs.

Reply:
No, please see the code 
https://github.com/ultraio/eosio.contracts/blob/master/contracts/eosio.nft.ft/src/nft.buy_offer.cpp#L279

Reply:
From the code https://github.com/ultraio/eosio.contracts/blob/master/contracts/eosio.nft.ft/src/nft.buy_offer.cpp#L279
```
ASSERTION_CHECK(row.factory_ids.size() + row.nft_ids.size() < config.max_active_offer_per_user, buyer_already_made_max_number_of_offers);
```
`max_active_offer_per_user` applies to the sum of offers made on Uniqs and offers on factories.

Reply:
So the description should be
An account will not be able to make more offers on either factories or Uniqs

Reply:
I was looking at the wrong line.
updated for this and staging doc to
`An account will not be able to make Uniq offers more than or equal to ...`